src/main/java/io/airlift/slice/XxHash64.java

```diff
@@ -56,7 +56,7 @@ public static long hash(long seed, Slice data, int offset, int length)
         checkPositionIndexes(0, offset + length, data.length());
 
         Object base = data.getBase();
-        final long address = data.getAddress() + offset;
```

Comment:
This was intentional, to make sure the address variable is not inadvertently overwritten as the code walks loops through the data.